features/steps/test_post_endpoint.rb

```diff
@@ -1,14 +1,38 @@
 class Spinach::Features::TestPostEndpoint < Spinach::FeatureSteps
   step "I create an Item" do
-    HTTParty.post("http://localhost:3000/api/v1/items", {body: {name: "Foo", quality: 4, sellIn: 4}})
+    credentials_manager = Api::V1::CredentialsManager.new
```

Comment:
Recommendation for Refactoring: This is great, but you can probably extract the `credentials_manager` and `credentials` into a `before_each` block.